examples/dataflow-streaming-schema-handler/README.md

```diff
@@ -0,0 +1,113 @@
+# Dataflow Streaming Schema Handler
+
+This package contains a set of components required to handle unanticipated incoming streaming data into BigQuery with schema mismatch.
+The code will uses Schema enforcement and DLT (Dead Letter Table) approach to store schema incompability. In case of schema incompability detected from the incoming message, the incoming message will be evolved to match the targeted table schema by adding and removing fields which then will be store in the targetted table; In addition, the original message (with schema incomability) will be store in the DLT for debugging and record purposes.
```

Comment:
lowercase after the semicolon:  `table; in addition` (or maybe a period after `table`).